src/constants/metricNames.ts

```diff
@@ -0,0 +1,3 @@
+export const METRIC_NAMES = {
+  viewHome: "loaded screen: home",
```

Comment:
@shanzzam Let me know if you would like to change this label! This is the event name that will show up in Amplitude

Reply:
This is fine. I guess as there's only one page we're ever tracking this works! We can add metrics for how the rest of the content is accessed via keypair selection later.